Adding YMDK60 HHKB PCB (#21004)
qmk/qmk_firmware: keyboards/ymdk/ymd60/readme.md

# YMDK 60 HHKB

> ![YMDK60 HHKB PCB](https://imgur.com/a/wuQ5XAG)

Reviewer: ```suggestion
![YMDK60 HHKB PCB](https://i.imgur.com/5lRZfcgh.png)
```

> A 65% PCB, which controlled by an ATMega32U4 chipset.

Reviewer 2: ```suggestion
A 60% PCB, which is controlled by an ATMega32U4 chipset.
```

* Keyboard Maintainer: [Pangorin](https://github.com/pangorin)

> * Hardware Supported: ATMega32U4-MU

Reviewer 3: ```suggestion
* Hardware Supported: ATMega32U4-MU
* Hardware Availability: [YMDK](https://ymdkey.com/products/ymdk-hhkb-layout-qmk-via-programmable-underglow-rgb-aluminum-cnc-sandwich-case-plate-mechanical-keyboard-kit)
```

A link to the vendor's website is a good way to verify that the firmware is compatible with the hardware _(unless there's a good reason for omitting this bullet point from the template)_.

Author: Nice, I commited your suggestion!

# YMDK 60 HHKB

![YMDK60 HHKB PCB](https://i.imgur.com/5lRZfcgh.png)

A 60% PCB, which is controlled by an ATMega32U4 chipset.

* Keyboard Maintainer: [Pangorin](https://github.com/pangorin)

> * Hardware Supported: ATMega32U4-MU

Reviewer 4: ```suggestion
* Hardware Supported: YMDK60 HHKB PCB
```
